Edits for topics in the Installing and Upgrading branches

francoughlin opened the pull request:
## What Changed?

Edits for topics in the Installing and Upgrading branches; some additional restructuring for the Linux install details section

nidhibhammar commented on product_docs/docs/epas/15/installing/linux_install_details/managing_an_advanced_server_installation/connecting_to_epas.mdx:
````diff
@@ -0,0 +1,28 @@
+Use the following command and options to start the `edb-psql` client:
+
+```text
+psql -d edb -U enterprisedb
````
```suggestion
edb-psql -d edb -U enterprisedb
```

nidhibhammar commented on product_docs/docs/epas/15/installing/linux_install_details/managing_an_advanced_server_installation/modifying_the_data_directory_location.mdx:
```diff
@@ -0,0 +1,128 @@
+
+## On RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x
+
+On a RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x host, the unit file is named `edb-as-15.service` and resides in `/usr/lib/systemd/system`. The unit file contains references to the location of the EDB Postgres Advanced Server (EPAS) `data` directory. Avoid making any modifications directly to the unit file because they might be overwritten during package upgrades.
```
I think we can make this generic by writing <x> in place of the exact version number across the topic. Thoughts?
```suggestion
On a RHEL/CentOS 7.x or RHEL/Rocky Linux/AlmaLinux 8.x host, the unit file is named `edb-as-<x>.service` and resides in `/usr/lib/systemd/system`. The unit file contains references to the location of the EDB Postgres Advanced Server (EPAS) `data` directory. Avoid making any modifications directly to the unit file because they might be overwritten during package upgrades.
```

nidhibhammar commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/invoking_the_graphical_installer_from_the_command_line/reference_command_line_options.mdx:
```diff
@@ -7,7 +7,7 @@ redirects:
 <div id="reference_command_line_options" class="registered_link"></div>
 
-You can optionally include the following parameters for an EDB Postgres Advanced Server installation on the command line or in a configuration file when invoking the EDB Postgres Advanced Server installer.
+You can optionally include the following parameters for an EDB Postgres Advanced Server (EPAS) installation on the command line or in a configuration file when invoking the EPASinstaller.
```
```suggestion
You can optionally include the following parameters for an EDB Postgres Advanced Server (EPAS) installation on the command line or in a configuration file when invoking the EPAS installer.
```

nidhibhammar commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/invoking_the_graphical_installer_from_the_command_line/reference_command_line_options.mdx:
```diff
@@ -31,15 +31,15 @@ Helps with troubleshooting installation problems. `debug_log` is the name of the
 `--disable-components component_list`
 
-Specifies a list of EDB Postgres Advanced Server components to exclude from the installation. By default, `component_list` contains '' (the empty string). `component_list` is a comma-separated list containing one or more of the following components:
+Specifies a list of EPAScomponents to exclude from the installation. By default, `component_list` contains '' (the empty string). `component_list` is a comma-separated list containing one or more of the following components:
```
```suggestion
Specifies a list of EPAS components to exclude from the installation. By default, `component_list` contains '' (the empty string). `component_list` is a comma-separated list containing one or more of the following components:
```

nidhibhammar commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/invoking_the_graphical_installer_from_the_command_line/reference_command_line_options.mdx:
```diff
@@ -31,15 +31,15 @@ Helps with troubleshooting installation problems. `debug_log` is the name of the
 `--enable_acledit { 1 | 0 }`
 
-The `--enable_acledit 1` option instructs the installer to grant permission to the user specified by the `--serviceaccount` option to access the EDB Postgres Advanced Server binaries and `data` directory. By default, this option is disabled if `--enable_acledit 0` is specified or if the `--enable_acledit` option is completely omitted.
+The `--enable_acledit 1` option instructs the installer to grant permission to the user specified by the `--serviceaccount` option to access the EPASbinaries and `data` directory. By default, this option is disabled if `--enable_acledit 0` is specified or if the `--enable_acledit` option is completely omitted.
```
```suggestion
The `--enable_acledit 1` option instructs the installer to grant permission to the user specified by the `--serviceaccount` option to access the EPAS binaries and `data` directory. By default, this option is disabled if `--enable_acledit 0` is specified or if the `--enable_acledit` option is completely omitted.
```

nidhibhammar commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/invoking_the_graphical_installer_from_the_command_line/reference_command_line_options.mdx:
```diff
@@ -52,15 +52,15 @@ Although this option is listed when you run the installer with the `--help` opti
 `--extract-only { yes | no }`
 
-Include the `--extract-only` parameter to indicate that the installer should extract the EDB Postgres Advanced Server binaries without performing a complete installation. Superuser privileges aren't required for the `--extract-only` option. The default value is `no`.
+Include the `--extract-only` parameter to indicate that the installer should extract the EPASbinaries without performing a complete installation. Superuser privileges aren't required for the `--extract-only` option. The default value is `no`.
```
```suggestion
Include the `--extract-only` parameter to indicate that the installer should extract the EPAS binaries without performing a complete installation. Superuser privileges aren't required for the `--extract-only` option. The default value is `no`.
```

francoughlin commented:
Hello, @nidhibhammar! I think we have addressed all of the comments, except for the one about the version number, which we will address as a separate ticket. Do you think we are OK to approve and merge? Thanks!

drothery-edb commented on product_docs/docs/epas/15/installing/uninstalling/linux_uninstall.mdx:
````diff
@@ -51,25 +51,26 @@ To uninstall EDB Postgres Advanced Server and its dependent packages, use the ap
-    yum remove edb-as15-server*
+    yum remove edb-as<xx>-server*
     ```
+Where <*xx*> represents the EDB Postgres Advanced Server version number.
````
left angle brackets outside of code formatting cause the build to fail. Options include enclosing <*xx*> in tick marks to format it as code, remove the angle brackets, or use the ASCII code for the left angle bracket (&lt;)
```suggestion
Where <*xx*> represents the EDB Postgres Advanced Server version number.
```

francoughlin replied:
Thanks for pointing that out! I checked out a couple of other products, and there isn't a consistent approach. Some use <x>, some <xx>, and some <X>. I think since we have a two-digit version number we should use `<xx>`. I checked with Betsy, and she told me to check with you guys :-)

drothery-edb replied:
I agree. Let's use `<xx>` like we do in the EPAS install topics:
![image](https://github.com/EnterpriseDB/docs/assets/83650384/05b8d2c4-5727-49a5-8621-0db82fb774ef)

francoughlin replied:
Will do! Thanks!

drothery-edb commented on product_docs/docs/epas/15/installing/uninstalling/windows_uninstall.mdx:
````diff
@@ -18,8 +18,9 @@ The EDB Postgres Advanced Server interactive installer creates an uninstaller th
-    uninstall-edb-as15-server.exe
+    uninstall-edb-as<xx>-server.exe
     ```
+    Where <*xx*> represents the EDB Postgres Advanced Server version number.
````
see earlier comment

drothery-edb commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/invoking_the_graphical_installer_from_the_command_line/performing_an_unattended_installation.mdx:
````diff
@@ -20,10 +20,12 @@ You must have administrative privileges to install EDB Postgres Advanced Server
 database_superuser_password --servicepassword system_password
 ```
 
+Where <*xx*> represents the EDB Postgres Advanced Server version number.
````
ditto

drothery-edb commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/performing_a_graphical_installation_on_windows.mdx:
```diff
@@ -7,9 +7,11 @@ redirects:
-To invoke the wizard, you must have administrator privileges. Assume administrator privileges, and double-click the `edb-as15-server-15.x.x-x-windows-x64` executable file.
+To invoke the wizard, you must have administrator privileges. Assume administrator privileges, and double-click the `edb-as<*xx*>-server-xx.x.x-x-windows-x64` executable file.
+
+Where <*xx*> represents the EDB Postgres Advanced Server version number.
```
ditto

drothery-edb commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/performing_a_graphical_installation_on_windows.mdx:
````diff
@@ -37,8 +39,9 @@ The `Installation Directory` window opens.
-C:\Program Files\edb\as15
+C:\Program Files\edb\as<*xx*>
 ```
+Where <*xx*> represents the EDB Postgres Advanced Server version number.
````
ditto

drothery-edb commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/performing_a_graphical_installation_on_windows.mdx:
````diff
@@ -85,26 +88,27 @@ After selecting the components you wish to install, click `Next` to open the `Ad
-C:\Program Files\edb\as15\data
+C:\Program Files\edb<*xx*>\data
 ```
+Where <*xx*> represents the EDB Postgres Advanced Server version number.
````
ditto

drothery-edb commented on product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/performing_a_graphical_installation_on_windows.mdx:
```diff
@@ -85,26 +88,27 @@ After selecting the components you wish to install, click `Next` to open the `Ad
 <div style="text-align: center"> Fig. 6: The EDB Postgres Advanced Server Dialect window </div>
 
-Use the drop-down listbox on the `EDB Postgres Advanced Server Dialect` window to choose a server dialect. The server dialect specifies the compatibility features supported by EDB Postgres Advanced Server.
+Use the drop-down listbox on the `EDB Postgres Advanced Server Dialect` window to choose a server dialect. The server dialect specifies the compatibility features supported by EPAS.
```
This PR has some EDB Postgres Advanced Server > EPAS changes also.

francoughlin replied:
Gotcha, thanks!
